db/migrate/20160808002834_add_files.rb

```diff
@@ -0,0 +1,29 @@
+class AddFiles < ActiveRecord::Migration
+  def change
+    create_table "katello_files" do |t|
+      t.timestamps
+      t.string "uuid", :null => false, :limit => 255
+      t.string 'name', :limit => 255
+      t.string 'checksum', :limit => 255
```

Comment:
I was following the other content types that were set to this based on (https://github.com/Katello/katello/pull/5837). If you think I should open this up to unlimited then I shall.